docs/source/notebooks/00_Introduction_to_JupyterLab.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>you can simply add a cell and "pip install <strong>package</strong>" as required</p><p> </p><p>It should be "! pip install <strong>package"</strong> to execute code in a cell.</p><p> </p><p>However use can also use the jupyter conda plugin to install via conda (Settings -&gt; Conda Packages Manager).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>thanks, added!</p>